### DISCUSSION TITLE
I'm wondering for TFT_eSPI.cpp

### DISCUSSION THREAD

**Discussion by stef-ladefense** [Bodmer/TFT_eSPI#2373] [Q&A - General] [answered]
hi,
why twice for : void TFT_eSPI::setTextFont(uint8_t f) ?
lines 5816 & 5841

in the setCursor with 3th param (font) line 2835
why no zero protection like setTextFont?
textfont = (f > 0) ? f : 1; // Don't allow font 0

merci

**Comment by Bodmer** [comment, marked as answer]
That satisfies the compiler when certain fonts are not included, see #if/#else conditions.
